Utils/Dataflow/pyDKB/common/hdfs.py

```diff
@@ -40,7 +49,17 @@ def check_stderr(proc, timeout=None, max_lines=1):
 
 
 def makedirs(dirname):
-    """ Try to create directory (with parents). """
+    """ Try to create directory (with parents).
+
+    Create directory and check status of process
+    with subprocess.Popen.
+
+    :param dirname: a name of a created directory
```

Comment:
Maybe "name of the directory" or something else?
1) "a" before "name" - similar instances of "name" in this file come without "a".
2) "a" before "directory" - it's not some directory, it's a specific directory which was already mentioned in lines 52 and 54. By the way, in those lines it can be "create a directory".
3) "created" - we are talking about arguments, so nothing was created yet. "to create" or "to be created" can be used if necessary.